hera_sim/visibilities/vis_cpu.py

```diff
@@ -90,11 +101,14 @@ def get_eq2tops(self):
         coords to topocentric at each LST.
         """
 
-        sid_time = np.unique(self.uvdata.lst_array)
+        sid_time = self.uvdata.lst_array[::self.uvdata.Nbls]
         eq2tops = np.empty((len(sid_time), 3, 3), dtype=self._real_dtype)
-
+        
+        #ra, dec = self.point_source_pos.T
+        
+        
         for i, st in enumerate(sid_time):
-            eq2tops[i] = conversions.eq2top_m(-st, self.uvdata.telescope_lat_lon_alt[0])
+            eq2tops[i] = conversions.eq2top_m(st, self.uvdata.telescope_lat_lon_alt[0]) ######### WAS -st
```

Comment:
can you go through the math and show that it should be +st here?

Reply:
```python
def eq2top_m(ha, dec):
    """
    Return the 3x3 matrix converting equatorial coordinates to topocentric
    at the given hour angle (ha) and declination (dec).
```
https://en.wikipedia.org/wiki/Hour_angle#Relation_with_the_right_ascension
![image](https://user-images.githubusercontent.com/32600543/64370884-3292ab00-cfd4-11e9-8538-fa21ea376581.png)

This is how I understand it, correct me where I'm wrong:
(1) The hour angle is passed as the first argument. (2) The conversion on Wikipedia is hour angle = lst - (right ascension of object) (3) the variable `st` is the lst so it is not negated?

Reply:
Although that would beg the question of why we wouldn't be taking into account the right ascension of the object? I do think I am misunderstanding something here.

Reply:
Yeah, I'm also confused, and I can't remember my thinking. Can you put a comment something like 

```
# TODO: ensure that LST is converted to HA correctly (https://en.wikipedia.org/wiki/Hour_angle#Relation_with_the_right_ascension)
```
above the call? 

We'll then deal with it once merged.